zp-core/zp-extensions/user-expiry.php

```diff
@@ -250,6 +250,7 @@ static function checklogon($loggedin, $user) {
 	 * @return string
 	 */
 	static function reverify($path) {
+		global $_zp_current_admin_obj;
 		//process any verifications posted
 		if (isset($_GET['user_expiry_reverify'])) {
 			$params = sanitize(unserialize(pack("H*", trim($_GET['user_expiry_reverify']), '.'), ['allowed_classes' => false]));
```

Comment:
This seems unnecessary as I cannot see the global being used within this method. But it does not hurt…

Reply:
Here
[268: $userobj->setLastChangeUser($_zp_current_admin_obj->getUser());](https://github.com/bic-ed/zenphoto/blob/813293227b2e68e07ceabfa44970447c794c5aa6/zp-core/zp-extensions/user-expiry.php#L268)

Reply:
Ah, sorry, you are totally right… (looked at another release stream where this was missing)